src/lib/config/arbitrum.json

```diff
@@ -37,6 +37,10 @@
     "lidoRelayer": "",
     "balancerHelpers": "0x77d46184d22CA6a3726a2F500c776767b6A3d6Ab"
   },
+  "keys": {
+    "infura": "daaa68ec242643719749dd1caba2fc66",
+    "alchemy": "ODJ9G5Ipv-Hb2zTWMNbUFIqv9WtqBOc2"
+  },
```

Comment:
Why do we need the keys in the file here? For defaults? We could potentially move these to ENV vars in Vercel if that helps?

Reply:
Yea it was inconsistencies between environment configs, different environments had different keys for the same services.